e2e/attributes.bats

```diff
@@ -143,22 +143,22 @@ teardown_file() {
   done
 
   run_otdfctl_attr list --limit 2
-    assert_success
-    assert_line --regexp "Current Offset.*0"
-    assert_line --regexp "Next Offset.*2"
-  
+  assert_success
```

Comment:
Did you find a linter for BATS tests? I generally find indented assertions on a single run more readable, but the indentations haven't been uniformly applied, and if this is auto-linted that would definitely be good to add to CI.